sysutils/smart/src/www/diag_smart.php

```diff
@@ -302,6 +304,9 @@ function add_colors($string)
       </section>
 
     <?php
+    } else {
+        echo "No SMART devices.";
```

Comment:
all output most be inside a gettext call (for translation)

Reply:
Sure thing. There are similar error messages elsewhere in this file, should I wrap them in gettext as well?
* echo "Device does not exist, bailing.";
* echo "Invalid test type, bailing.";